apps/desktop/src/app/sync/sources/sitemap.ts

```diff
@@ -29,15 +28,11 @@ class SitemapImpl implements ISourceConnector {
   }
 
   handleParameters(params: ConnectorParameters) {
-    if (params['url']) {
-      this.url = params['url'];
-    }
+    // eslint-disable-next-line no-empty-function
   }
 
   getParametersValues(): ConnectorParameters {
-    return {
-      path: this.url,
-    };
+    return {};
```

Comment:
This looks wrong, `getParametersValues` is supposed to return all the config values that will be useful in the sync server, so obviously, we need the sitemap url if we want the server to be able to download it.
Nevertheless, in the `main` branch I see we return `path`, while in the sitemap connector we try to read `sitemap`: ```this.params['sitemap']```

So something is also probably wrong in the current implementation.